src/app/projects/new-project/new-project.component.ts

```diff
@@ -38,7 +43,8 @@ export class NewProjectComponent implements OnInit {
     this.projectsService.addProject(this.project)
       .subscribe(response => {
         this.project = response;
-        this.router.navigateByUrl('/projects');
+        this.showConfirm = true;
+        setTimeout(() => this.router.navigateByUrl('/projects'), 2000);
```

Comment:
a czemu akurat po 2000?

Reply:
taka wartość mi się wylosowała xd
możesz zaproponować inną.

Reply:
brak wartości?

Reply:
no ale wtedy nie zobaczę `confirmMessage`.
jak się nauczę robić tosty, to się zgodzę na brak wartości.
mam przerabiać na tosty?

Reply:
Ja bym głosowała za tym, żeby ograniczyć dzisiaj zdobywanie nowych umiejętności - chyba że da się skądś ukraść tosty

Reply:
wstrzykujesz ```private messageService: MessageService```, w html trzeba dodać ```<p-toast></p-toast>```, wywołanie: 
```
 this.messageService.add({
          severity: 'error',
          summary: '123',
          detail: '123'
        });
```

Reply:
dziękuję, już umiem. podwaliłem z Twojego logowania @szbrnl